src/content/certifications/exam-faqs/vault-associate-003.mdx

```diff
@@ -0,0 +1,123 @@
+<!-- Each "## Heading Two" section in this document will be used to populate an FAQ item -->
+
+## Exam Availability
+Coming soon!
+
+## Which Exam to Take
+You can use either exam to validate Vault knowledge at the associate level. The [Vault Associate 002 certification](/certifications/security-automation#vault-associate-002-details) is still relevant and will be accepted as validation of Vault knowledge until the badge’s expiration date. You can hold both the Vault Associate 002 and Vault Associate 003 at the same time.
+
+## Content Differences Between Exam Versions
+We updated the Vault Associate 003 exam to account for how Vault has grown, and to accommodate future growth. The changes are primarily a reorganization and rewording of the 002 exam objectives. More significant changes are listed below.
+
+The Vault Associate (003) tests on Vault version 1.16 and now includes HCP Vault content.
+
+|    | **NEW topics covered in (003)**                                         |
+| -- | ----------------------------------------------------------------------- |
+| 8e | Differentiate between self-managed and HashiCorp-managed Vault clusters |
+| 9b | Describe the Vaults Secrets Operator                                    |
```

Comment:
This is missing the second table that was on the asana card https://app.asana.com/0/1207927636763096/1208183949346469 is that on purpose?

Reply:
Good catch. I think we should add it. Let me do that and push

Reply:
This is now added and ready for review